lib/heimdall_tools/nessus_mapper.rb

```diff
@@ -107,7 +120,29 @@ def finding(issue, timestamp)
       [finding]
     end
 
-    def nist_tag(pluginfamily, pluginid)
+    def read_cci_xml
+      cci_list_path = File.join(File.dirname(__FILE__), '../data/U_CCI_List.xml')
```

Comment:
See other comment: @rbclark and @Bialogs - does this work to ensure that we will always have access to the data file even when we gem install?